src/test/java/homework_4/singleton/SingletonTest.java

```diff
@@ -0,0 +1,16 @@
+package homework_4.singleton;
+
+import org.junit.jupiter.api.Test;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+class SingletonTest {
+
+    @Test
+    void getInstance() {
+        Singleton singleton1 = Singleton.getInstance();
+        Singleton singleton2 = Singleton.getInstance();
+
+        assertEquals(singleton1, singleton2);
```

Comment:
Opt: they can be equal, but be different objects, right?

Reply:
they share a common pointer. (added this check (IDE don't like this))